src/main/java/frc/robot/constants/ArmConstants.java

```diff
@@ -5,7 +5,7 @@
 
 public class ArmConstants {
 
-    public static final int MOTOR_ID = 1;
+    public static final int MOTOR_ID = 0;
```

Comment:
Why are you changing arm constants in this PR?

Reply:
Because those don't allow the drivetrain to run at all so main doesn't work. We might as well change this in this PR